dashboard/app/controllers/registrations_controller.rb

```diff
@@ -91,7 +91,8 @@ def update_params(params)
       :user_type,
       :school,
       :full_address,
-      :terms_of_service_version
+      :terms_of_service_version,
+      races: []
```

Comment:
Looks like you actually have to pass an empty array as a parameter in order to whitelist array elements